app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::UserController < ApplicationController
+
+	def index
+		@users = User.includes(:recipes).all
+		render json: @users, :include => [:recipes]
+	end
+end
```

Comment:
Looks like this file is using tabs instead of spaces. Let's stick to two spaces as a tab.

Let's also ensure every file ends w/ a newline.

Reply:
Looks great.
I would agree with Henry that two spaces is conventionally used in Ruby and would keep things consistent.